app/screens/LoginScreen.tsx

```diff
@@ -208,6 +218,7 @@ const LoginScreen = () => {
                   name="userName"
                   label={t("model.user.legacy_address")}
                   returnKeyType="next"
+                  valueHook={addressValueHook}
```

Comment:
Why not simply assign it to the onChange / onChangeText param?

Reply:
also name is _a bit_ misleading (in a react project) 😝

Reply:
Because it is not available on our `Input`, but `valueHook` was

Reply:
Ah, not extended actually - got it
Ok, a bit weird implementation/abstraction 😄